verticapy/tests/vModel/test_decision_tree_regressor.py

```diff
@@ -231,6 +231,15 @@ def test_to_sklearn(self, base):
 
         model_sk.drop()
 
+    def test_to_python(self, model):
+        model.cursor.execute(
+            "SELECT PREDICT_RF_REGRESSOR(* USING PARAMETERS model_name = '{}', match_by_pos=True)::float FROM (SELECT 'Male' AS \"Gender\", 0 AS \"owned cars\", 'Cheap' AS \"cost\", 'Low' AS \"income\") x".format(
```

Comment:
You can simplify the query into:
"SELECT PREDICT_RF_REGRESSOR('Male', 0, 'Cheap', 'Low' USING PARAMETERS model_name = '{}', match_by_pos=True)::float"

Reply:
Done;